packages/react-native-web/src/vendor/react-native/VirtualizedList/index.js

```diff
@@ -1703,6 +1716,11 @@ class VirtualizedList extends React.PureComponent<Props, State> {
               this._getFrameMetricsApprox,
               this._scrollMetrics,
             );
+
+            // revert the state if calculations are off
```

Comment:
When would this happen  / I think a more thorough comment here would be helpful

Reply:
added a comment here. 

```
// revert the state if calculations are off
// this would only happen on the inverted flatlist (probably a bug with overscroll-behavior)
// when scrolled from bottom all the way up until onEndReached is triggered
```